elephant/spade.py

```diff
@@ -1041,27 +1209,51 @@ def test_signature_significance(pvalue_spectrum, alpha, corr='', report='#'):
         tests = x_array[:, -1] <= alpha * 1. / len(pvalue_spectrum)
     elif corr in ['f', 'fdr']:  # or with FDR correction
         tests, pval, rank = _fdr(x_array[:, -1], alpha=alpha)
+    elif corr in ['hb', 'holm_bonf']:
+        tests = _holm_bonferroni(x_array[:, -1], alpha=alpha)
     else:
-        raise AttributeError("corr must be either '', 'b'('bonf') or 'f'('fdr')")
-
+        raise AttributeError(
+            "corr must be either '', 'b'('bonf') or 'f'('fdr')")
```

Comment:
Is `corr` a parameter?

Reply:
Yes it is, should I make this explicit?

Reply:
Yes, just for readability would be nice to have e.g. "Parameter `corr` must be ..."

Reply:
Done!